src/app/widgets/comment-module/comment.component.less

```diff
@@ -1,6 +1,6 @@
 @import (reference) '../../../assets/stylesheets/_base';
 
-.f8-comment {
+.f8-planner__comment {
```

Comment:
This should be `f8-planner-comment`. Although make sure it doesn't conflict with the other comment class. This is why naming blocks based on the component name is a good practice to follow.